templates/admin/login/login.html.twig

```diff
@@ -1,7 +1,7 @@
 {% extends '@PimcoreAdmin/admin/login/layout.html.twig' %}
 
 {% block content %}
-<div id="loginform">
+<div id="loginform" {% if not browserSupported %}style="display:none;"{% endif %}>
```

Comment:
@NiklasBr Why do you move the css to the inline style?

Reply:
![image](https://github.com/pimcore/admin-ui-classic-bundle/assets/279826/330affff-b13f-4e82-90f2-2c39c787ab6a)

Reply:
@NiklasBr What is your opinion about adding this here? https://github.com/pimcore/admin-ui-classic-bundle/pull/227/files#diff-55e6dc29c563c3b12119df18461f947de2b3c73c6dae2cd25a0a2400f0e64ca6L52

Reply:
I think it spreads out the logic in an unnecessary way, but that is not a strongly held opinion.

Reply:
We have the JS part also there, do you see any reason why this shouldn't be together?

Reply:
Because this PR was meant to only fix validation errors and not become an everythingburger :)

Reply:
I totally agree with you, but I don't get the difference of change between moving it to the inline styling or moving it to the `below_footer` block. So you basically need to move it with both approaches, right?

Reply:
@robertSt7 `below_footer` is also inside of the body tag. You would need a new block for the head section.

Reply:
@blankse You are right. Thanks 
@NiklasBr Then we would consider this in a follow up PR